service/service.go

```diff
@@ -153,3 +131,40 @@ func (srv *service) Shutdown(ctx context.Context) error {
 	// TODO: Shutdown TracerProvider, MeterProvider, and Sync Logger.
 	return errs
 }
+
+func (srv *service) initPipeline(set *settings) error {
```

Comment:
This also initialize the extensions. Maybe have rename to `initExtensionsAndPipelines`?